xoinvader/enemy_wave.py

```diff
@@ -0,0 +1,78 @@
+"""Module for creating and maintaining waves of incoming enemies."""
+
+class EnemyWave(object):
+    """Container for enemy wave behavior.
+
+    Intended to use as just container, but may be subclassed. It's useful in
+    order to add methods for creating concrete animations and spawning actual
+    enemies.
+
+    Spawned enemies are tracked automatically and properly removed when
```

Comment:
Yep, indeed.
I thought the same thing and proceeded to documenting other parts of the file.

So I dunno, you say me! :wink:
Anyway, enemies are not tracked, so the docstring should be rewritten.